pkg/controller/volume/persistentvolume/controller.go

```diff
@@ -964,7 +964,7 @@ func (ctrl *PersistentVolumeController) recycleVolumeOperation(arg interface{})
 	volume = newVolume
 
 	// Find a plugin.
-	spec := vol.NewSpecFromPersistentVolume(volume, false)
+	spec := vol.NewSpecFromPersistentVolume(volume, false, "")
```

Comment:
I don't like the namespace being left empty unless absolutely necessary. Can the namespace be passed in here from the PVC or something?